### PR TITLE
Chloe create

### PR DESCRIPTION
**Pull request by chloemackallah**
I've written as much as I could find time for. Hopefully this is a good start, and sets the tone and scope for the create section.
Fingers crossed I am able to continue it in December!

### CODE REVIEW

**Comment by paolap** [comment]
Thanks @chloemackallah, from my point of view this can be merged as it is, then we can start breaking down the 3 main sections into separate files, so they also appear as such in the left hand panel.
So create a file each for these:
1) Dataset creation basics & sharing recommendations
2) New, modified, and derived datasets
3) Requirements for publication

If we start filling in the details then it will be easier to see if we want to make changes to the structure/titles and how they fit in with the other guidelines. Thanks for getting this done!

**Review by paolap — APPROVED** [review]
On closer inspection, I noticed that the new files were't included in the index _toc.yml file, so I added them and pushed the branch with the new _toc.yml. And also just one comment on the publish section, we can link the jpourn al requirements, but they're usually do not touch on file creation at all, so it might be irrelevant in this section, all we can do is to recommend to still create sensible files even if there's no requirements. No need to make that change now, I'm still happy to approve, merge and then work on the details :-)

File: Governance/create/create-publishing.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Requirements for publication
+
+## Publishing in a journal
+
+Move journal requirements here? https://acdguide.github.io/Governance/concepts/journal.html
```
Comment (hot007) [inline Governance/create/create-publishing.md:5]: I would simply say something like "Many journals require data (and sometimes code) be published accompanying a manuscript. See <link to concepts/journal> for examples."

File: Governance/create/create-basics.md [hunk 2]
```diff
@@ -2,62 +2,67 @@
...
+Climate data is a highly specialised field of data science, due to the size and complexity which often require computing and scientific coding skills, the variety of metadata fields required to adequately describe the data and its dimensions, the domain-specific scientific knowledge required to make informed decisions about its creation and use, and the technical knoweldge required to produce robust datasets that can be reused by others. (Note that we will use terms and concepts described in [Appendix/concepts](appendix/concepts.md).)
 
-Chloe:  
+By far the most commonly used format in the climate science community is [netCDF](https://www.unidata.ucar.edu/software/netcdf/), a self-describing (i.e. metadata is an in-built feature) array-oriented (i.e. highly structured data such as climate model data) format that is typically used on POSIX systems (Unix-based computing system with a directory structure and command-line input; standard for high-performance computing systems such as [NCI](https://nci.org.au/)). See the [technical note on data formats](tech/data_formats.md).
```
Comment (hot007) [inline Governance/create/create-basics.md:10]: Add "open" in here somewhere (ie it's not a proprietary format)

File: Governance/create/create-basics.md [hunk 3]
```diff
@@ -2,62 +2,67 @@
...
+* Global attributes are key:value pairs that descibe the file at the top-level. While these are typically chosen according to the use case of the data and can vary significantly between modelling realms or scientific need, standards also exist for these. Common global attributes include dataset title, provenance information (i.e. where the data came from), license, and contact information, as well as naming any metadata conventions implemented in the file.
 
-Keywords: https://acdguide.github.io/Governance/tech/keywords.html
+It is likely that the raw scientific data that you are building your datasets around are already highly structured (even in netCDF format already), so your main effort here will be ensuring that the metadata correctly and adequately described your data. 
 
-CVs: https://acdguide.github.io/Governance/concepts/controlled-vocab.html, https://acdguide.github.io/Governance/concepts/conventions.html
+### Attributes
 
-Machine-readable metadata attributes (units, coordinates, time, fill_values)
+NetCDF attributes are in one of two categories: Machine-readable metadata attributes  and Human-readable metadata attributes (long names, most global attrs).
```
Comment (hot007) [inline Governance/create/create-basics.md:26]: Double space, "human" probably doesn't need a capital?

**Review by hot007 — APPROVED** [review]
Although I've made a couple of minor edit comments here, I concur with Paola, I'm happy to get this merged and then work on it.

**Comment by chloemackallah** [comment]
That's great, thanks Paola & Claire!
I'm going to go ahead and merge this :)

**Pull request merged**